README.md

```diff
@@ -80,17 +85,29 @@ according to your preferences.
 ### Configurable parameters
 
 **Maven** :
-<br >Path of the `mvn` executable allows Exhort to locate and execute the `mvn` commands to resolve dependencies for
+<br >Path of the `mvn` executable allows Exhort to locate and execute the `mvn` command to resolve dependencies for
```

Comment:
Suggest a minor rewrite here at the beginning of the sentence.

``Path of the `mvn` executable allows ...`` --> 
`Set the full path of the Maven executable, which allows ...`